pybaseball/lahman.py

```diff
@@ -8,11 +8,12 @@
 
 from . import cache
 
-url = "https://github.com/chadwickbureau/baseballdatabank/archive/master.zip"
-base_string = "baseballdatabank-master"
+url = "https://github.com/jmaslek/LahmanDatabase/archive/refs/heads/main/baseballdb.zip"
+base_string = "LahmanDatabase-main/baseballdb"
```

Comment:
@jmaslek I didn't see a `LahmanDatabase-main/baseballdb` in your github archive at https://github.com/jmaslek/LahmanDatabase/archive/refs/heads/main/baseballdb.zip

However, I did see a `LahmanDatabase-main/unzipped` directory there.  Should the base_string be `"LahmanDatabase-main/unzipped"` instead?
```suggestion
base_string = "LahmanDatabase-main/unzipped"